lib/config.ex

```diff
@@ -5,6 +5,19 @@ defmodule Xcribe.Config do
 
   @valid_formats [:api_blueprint, :swagger]
 
+  @doc """
+  Return if serve mode is enable.
```

Comment:
Return true if server mode is enable? WDYT?

Reply:
:+1: